client/styles/device.ts

```diff
@@ -2,7 +2,7 @@ export const width = Object.freeze({
   mobileS: '320px',
   mobileM: '375px',
   mobileL: '425px',
```

Comment:
I think having 2 mobile sizes with the same size is redundant. We can probably remove mobileL and keep mobile since mobile is used more frequently, i.e its the standard in our codebase